client/src/components/layout/DogShowPage.js

```diff
@@ -1,10 +1,15 @@
 import React, { useState, useEffect } from "react";
 import Link from "react-dom";
 
+import DogReviewForm from "./DogReviewForm";
+import Fetcher from "./services/Fetcher.js";
+
 const DogShowPage = (props) => {
   const [dog, setDog] = useState({});
+  const [reviews, setReviews] = useState([]);
```

Comment:
Where do these reviews come from? As in it doesn't look like we are displaying the reviews anywhere on the page? Is that a separate feature?

We would also assume that the reviews would come from the getDog fetch request, for the reviews about that dog.

Reply:
We are not displaying these reviews because we are waiting for the other group to finish on "listing existing reviews" on the right side of the show page. So that we can add to that list. We will add the functionality to add the new reviews to the list once we merge today